[main bbfbbda] Update
 3 files changed, 209 insertions(+), 2 deletions(-)
To https://github.com/iancharnas/clevelandlightning.git
   b72152b..bbfbbda  main -> main
[main fc87fc6] Update
 3 files changed, 13053 insertions(+)
To https://github.com/iancharnas/clevelandlightning.git
   bbfbbda..fc87fc6  main -> main
[main 9c55e7e] Update
 3 files changed, 49215 insertions(+), 1 deletion(-)
To https://github.com/iancharnas/clevelandlightning.git
   fc87fc6..9c55e7e  main -> main
[main 378e9cb] Update
 3 files changed, 73520 insertions(+)
 create mode 100644 data/2021-12-21.csv
To https://github.com/iancharnas/clevelandlightning.git
   9c55e7e..378e9cb  main -> main
[main 8e02f9d] Update
 3 files changed, 75356 insertions(+)
 create mode 100644 data/2021-12-22.csv
To https://github.com/iancharnas/clevelandlightning.git
   378e9cb..8e02f9d  main -> main
Traceback (most recent call last):
  File "scrapeBlitzortungLightningStrikes.py", line 67, in <module>
    # print(countries)
  File "scrapeBlitzortungLightningStrikes.py", line 26, in main
    [latitude, longitude, strike_time, server, mds, mcg, sta] = row
KeyboardInterrupt
Reading existing strikes...
Traceback (most recent call last):
  File "scrapeBlitzortungLightningStrikes.py", line 70, in <module>
    main()
  File "scrapeBlitzortungLightningStrikes.py", line 27, in main
    if strike_time not in recorded_times:
KeyboardInterrupt
Traceback (most recent call last):
  File "scrapeBlitzortungLightningStrikes.py", line 67, in <module>
    main()
  File "scrapeBlitzortungLightningStrikes.py", line 27, in main
    recorded_times.add(strike_time)
AttributeError: 'dict' object has no attribute 'add'
